atomic_reactor/plugins/post_koji_import.py

```diff
@@ -192,7 +192,7 @@ def set_media_types(self, extra):
 
         # Append media_types from verify images
         media_results = self.workflow.data.postbuild_results.get(PLUGIN_VERIFY_MEDIA_KEY)
-        if media_results:
+        if media_results and not isinstance(media_results, Exception):
```

Comment:
verify media plugin will run, and it should be required now: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/tasks/binary.py#L59

because now media_types depend on it, so if verify media plugin fails, build should fail as well, as we shouldn't have empty media_types in brew